opennlp-docs/src/docbkx/doccat.xml

```diff
@@ -127,11 +127,9 @@ $ opennlp DoccatTrainer -model en-doccat.bin -lang en -data en-doccat.train -enc
 		<programlisting language="java">
 						<![CDATA[
 DoccatModel model = null;
-InputStream dataIn = null;
-
-try (dataIn = new FileInputStream("en-sentiment.train")) {
+try {
```

Comment:
Is there any reason that you avoid using try-with-resources here? (and in postagger.xml)

Reply:
The original code put FileInputStream in try-with-resource portion, in the latest version PlainTextByLineStream, InputStream is not necessary, so I removed the FileInputStream as well as the try-with-resource portion.
I think it would be better if ObjectStream is in try-with-resource.

Reply:
Ah, yeah! It's not Closeable. You're right. :)